container-aws-python/__main__.py

```diff
@@ -0,0 +1,41 @@
+from pulumi import Config, Output, export
+import pulumi_aws as aws
+import pulumi_awsx as awsx
+
+config = Config()
+container_port = config.get_int("containerPort") or 80
+cpu = config.get_int("cpu") or 512
+memory = config.get_int("memory") or 128
```

Comment:
May I suggest using the built-in default value option, instead? It's a slight bit more idiomatic to match the built-in dictionary call:

```suggestion
container_port = config.get_int("containerPort", 80)
cpu = config.get_int("cpu", 512)
memory = config.get_int("memory", 128)
```